### PR TITLE
Cell visibility feature

### PR DESCRIPTION
**Pull request by stevemayhew**
As requested from comment on pull request #261 Here are methods to introspect visibility of cells and get the set of currently 'visible' cell elements.  This will support lazy-load visible and many other use cases.

### CODE REVIEW

File: js/flickity.js [hunk 1]
```diff
@@ -560,6 +560,67 @@ Flickity.prototype.getAdjacentCellElements = function( adjCount, index ) {
...
+ * @param cellIndex - index of the cell in the Flickity
+ * @returns {truthy} - that is false if not visible, truthy if visible
+ */
+Flickity.prototype.cellVisibility = function( cellIndex ) {
+  var value = false;
+  var cell = this.cells[ cellIndex ];
+  var cellViewportX;
+
+  if (cell) {
+    cellViewportX = cell.x + this.x;  // translate cell coordinates to viewport
```
Comment (stevemayhew) [inline js/flickity.js]: There might be a better choice then ```this.x``` as it is not stable during animation.  Happy to hear alternatives?

Comment (desandro) [inline js/flickity.js]: You can use `.getRestingPosition()`. See https://github.com/metafizzy/flickity/blob/v1.1.1/js/animate.js#L225

Comment (stevemayhew) [inline js/flickity.js]: Thanks, will do!

**Comment by desandro** [comment]
Thanks again for this contribution. I implemented a different solution for [Flickity v2's `groupCells` option](http://flickity.metafizzy.co/options.html#groupcells). I appreciate your effort here and hope you continue to get involved in open source!

**Pull request closed without merging**